src/core/include/openvino/core/parallel.hpp

```diff
@@ -751,6 +866,9 @@ void for_6d(const int& ithr,
 
 template <typename T0, typename T1, typename T2, typename T3, typename T4, typename T5, typename F>
 void parallel_for6d(const T0& D0, const T1& D1, const T2& D2, const T3& D3, const T4& D4, const T5& D5, const F& func) {
+    if (D0 == T0(0) || D1 == T1(0) || D2 == T2(0) || D3 == T3(0) || D4 == T4(0) || D5 == T5(0)) {
+        return;
+    }
```

Comment:
Consider instaed adding these checks, just check `work_amount` is zero ?

Reply:
That is not applicable for OV_THREAD_SEQ